docs/GCSMultiFiles-batchsink.md

```diff
@@ -67,6 +67,10 @@ When running on other clusters, the file must be present on every node in the cl
 The 'avro' supports 'snappy' and 'deflate'. The parquet supports 'snappy' and 'gzip'. 
 Other formats does not support compression.
 
+**Allow flexible schemas in Output**: When enabled, this sink will write out records with arbitrary schemas. 
```

Comment:
Why is this file being updated?

Reply:
The GCS MultiSink PR was merged earlier but at the time the docs were not updated.